.pylintrc

```diff
@@ -187,7 +187,7 @@ max-nested-blocks=5
 [FORMAT]
 
 # Maximum number of characters on a single line.
-max-line-length=80
+max-line-length=79
```

Comment:
Because [PEP-8](https://www.python.org/dev/peps/pep-0008/#id19) says 79 and pycodestyle checks it (with no compromises). It's in fact 80 including the '\n' symbol.